pages.de/common/npm.md

```diff
@@ -0,0 +1,36 @@
+# npm
+
+> Ein Kommandozeilenwerkzeug für die Verwaltung von Javascript und Node.js Paketen (Packages)
+> Mehr Informationen: <https://www.npmjs.com>.
+
+- Erstelle eine `package.json` Datei interaktiv:
+
+`npm init`
+
+- Installiere alle in der `package.json` Datei gelisteten Abhängigkeiten:
+
+`npm install`
+
+- Installiere eine spezifische Version eines Packages und füge es automatisch der `package.json` Datei hinzu:
+
+`npm install {{package_name}}@{{version}}`
+
+- Installiere ein Package und füge es als Entwicklungs Abhängigkeit der `package.json` Datei hinzu:
```

Comment:
I wasn't sure whether I should try to translate dev dependencies to German. What do you think? Should I try to avoid English words as much as possible?

Reply:
if you are talking about the `devDependencies` section in the `package.json` I think not.
but if you are talking about development dependencies then yes.

Reply:
I'd either use `Entwicklungs-Abhängigkeit` or `Entwicklungsabhängigkeit`.

Reply:
🤯 that is a really long word :D

Reply:
Nothing compared to `Grundstücksverkehrsgenehmigungszuständigkeitsübertragungsverordnung`. 😄
But that's why I suggested the semi-correct version with the hyphen.

Reply:
And you can actually read that like a normal word?

Reply:
@bl-ue Yes, that’s the German language in a nutshell.

Reply:
After a few tries.